_docs/extending-expired-ca-cert.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: Extending Expired Puppet CA Certificate
```

Comment:
I guess it apply the same way with OpenVox packages.  Someone suggested to use the wording "OpenVox / Puppet", which kinda make sense.  In a title I would remove the "/ Puppet" to keep it short.